src/i18n/es/index.js

```diff
@@ -0,0 +1,156 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  en: {
+    message: {
+      ...potato,
+      help: {
+        label: 'Abrir tutorial'
+      },
+      print: { ...print },
+      tester: { ...tester },
+      keyboard: {
+        label: 'teclado'
+      },
+      layout: {
+        label: 'plano'
+      },
+      keymapName: {
+        label: 'Nombre de keymap',
+        placeholder: 'nombre de keymap personalizado'
+      },
+      loadDefault: {
+        label: 'Cargar valor por defecto',
+        title: 'Cargar valor por defecto para plano de Firmware QMK'
+      },
+      compile: {
+        label: 'Compilar',
+        title: 'Compilar el keymap'
+      },
+      downloadKeymap: {
+        title: 'Descargar solamente keymap.c',
+        label: 'solo keymap'
+      },
+      downloadSource: {
+        title: 'Dsecargar código de Firmware QMK',
+        label: 'Código completo'
+      },
+      downloadJSON: {
+        title: 'Exportar archivo JSON de Keymap QMK',
+        label: 'Keymap.JSON'
+      },
+      importJSON: {
+        title: 'Importar archivo JSON de Keymap QMK'
+      },
+      importUrlJSON: {
+        title: 'Import archivo JSON de Keymap QMK de URL'
```

Comment:
```suggestion
        title: 'Importar archivo JSON de Keymap QMK desde URL'
```

Reply:
I've assumed that here what it means is that you provide an URL to load the JSON from so that is why I've suggested `desde`